plugins/inputs/slurm/README.md

```diff
@@ -0,0 +1,227 @@
+# SLURM Input Plugin
+
+This plugin gather diag, jobs, nodes, partitions and reservation metrics by
+leveraging SLURM's REST API as provided by the `slurmrestd` daemon.
+
+This plugin targets the `openapi/v0.0.38` OpenAPI plugin as defined in SLURM's
+documentation. That particular plugin should be configured when starting the
+`slurmrestd` daemon up. For more information, be sure to check SLURM's
+documentation [here][SLURM Doc].
+
+A great wealth of information can also be found on the repository of the
+Go module implementing the API client, [pcolladosoto/goslurm][].
+
+[SLURM Doc]: https://slurm.schedmd.com/rest.html
+[pcolladosoto/goslurm]: https://github.com/pcolladosoto/goslurm
+
+## Global configuration options <!-- @/docs/includes/plugin_config.md -->
+
+In addition to the plugin-specific configuration settings, plugins support
+additional global and plugin configuration settings. These settings are used to
+modify metrics, tags, and field or create aliases and configure ordering, etc.
+See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
+
+[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
+
+## Configuration
+
+```toml @sample.conf
+# Gather SLURM metrics
+[[inputs.slurm]]
+  ## Slurmrestd URL. Both http and https can be used as schemas.
+  url = "http://127.0.0.1:6820"
+
+  ## Credentials for JWT-based authentication.
+  # username = "foo"
+  # token = "topSecret"
+
+  ## Endpoints to skip gathering data from. By default all
+  ## the endpoints are queried for data.
+  # ignoreDiag = true
+  # ignoreJobs = true
+  # ignoreNodes = true
+  # ignorePartitions = true
+  # ignoreReservations = true
+
```

Comment:
In general we like to not have a lot of boolean config options. We would much rather have a string slice that lists what to excludue or include.

```toml
## Ignored endpoints
## List of endpoints a user can ignore, choose from: diag, jobs, 
##   nodes, partitions, reservations
# ignored_endpoints = []

Reply:
That should have been done on 2b0814c47b607ec0009e313358aefa2faa798189 😜